udata/core/dataset/forms.py

```diff
@@ -60,6 +61,12 @@ class CommunityResourceForm(BaseResourceForm):
     organization = fields.PublishAsField(_('Publish as'))
 
 
+def map_legacy_frequencies(form, field):
```

Comment:
To ensure backward compatibility: it allows to submit legacy frequencies, it's still working.
- avoid new version in API
- avoid to change tools using API

Was discussed before with @odtvince